test/app/lib/storage/encryptedStorage.spec.ts

```diff
@@ -12,18 +12,15 @@ const globalStorage = new EncryptedStorage(mockedBackend, encryptionSettings)
 
 describe("EncryptedStorage", () => {
   it("should put a string", async () => {
-    const result = await globalStorage.put("foo", "bar")
-    expect(result).toBeUndefined()
+    await globalStorage.put("foo", "bar")
   })
```

Comment:
I prefer to be explicit and specify what exactly the test case is checking, so imo I would prefer something like this:

```js
it("should put a string", () => {
    expect(async () => await globalStorage.put("foo", "bar")).not.toThrow()
})
```

Reply:
sorry, I got it wrong, revert my comment 😞

Reply:
No problem! This is reverted in #46